doc/plotting.rst

```diff
@@ -134,6 +134,42 @@ axes created by ``plt.subplots``.
 
 On the right is a histogram created by :py:func:`xarray.plot.hist`.
 
+.. _plotting.figsize:
+
+Controlling the figure size
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can pass a ``figsize`` argument to all xarray's plotting methods to
+control the figure size. For convenience, xarray's plotting methods also
+support the ``aspect`` and ``size`` arguments which control the size of the
+resulting image via the formula ``figsize = (aspect * size, size)``:
+
+.. ipython:: python
+
+    air1d.plot(aspect=2, size=3)
+    @savefig plotting_example_size_and_aspect.png
+    plt.tight_layout()
+
+.. ipython:: python
+    :suppress:
+
+    # create a dummy figure so sphinx plots everything below normally
+    plt.figure()
+
+This feature also works with :ref:`plotting.faceting`.
```

Comment:
I think `figsize` refers to the entire figure when used with faceting, but `size` only refers to a single panel. We should either make that consistent (both referring to single panels?) or note the difference.